lib/daylite/large_data.rb

```diff
@@ -0,0 +1,11 @@
+class Daylite::LargeData < Daylite::Base
+
+  self.table_name = "largedata"
+  
+  has_one :note, foreign_key: "largedataid"
+  has_one :organization, foreign_key: "detailsid"
+
+  def to_s
+    "#<Daylite::LargeData:#{plaintext}>"
```

Comment:
This format is more suitable for `#inspect`,  not `#to_s`. I would expect `to_s` to return plain text representation of the data, without its class name and module.

Reply:
Agreed.  This was a legacy method left over from the old code and serves no discernible purpose.